Analytics/Classes/Internal/SEGAnalyticsUtils.m

```diff
@@ -146,6 +146,7 @@ static id SEGCoerceJSONObject(id obj)
 
 static void AssertDictionaryTypes(id dict)
 {
+#ifdef DEBUG
```

Comment:
This should be `#if DEBUG`.
If DEBUG is defined as `0` this would evaluate as true :)

Reply:
Good point. However in practice that's not how most projects (including `Pods.xcodeproj`) are setup though.